tests/extras/test_neo4j.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from langroid.graph_database.neo4j import Neo4j, Neo4jConfig
+
+"""
+Before running this test case, make sure you run neo4j container using
```

Comment:
will you have a workflow to run this service, like we did for Meilisearch?

Reply:
I need to check `Meilisearch`.

Reply:
Yes, I'll do that, should I put the `test_neo4j.py` under the main folder? it's currently under extras.

Reply:
Yes extras